### PR TITLE
Allow users to specify whether to use inner product or L2 for buildin…

### PR DESCRIPTION
**Pull request by jonhilgart22**
…g faiss vectors

### CODE REVIEW

File: pyserini/index/faiss.py [hunk 1]
```diff
@@ -35,6 +35,7 @@
     parser.add_argument('--pq-m', type=int, default=192, required=False)
     parser.add_argument('--pq-nbits', type=int, default=8, required=False)
     parser.add_argument('--threads', type=int, default=12, required=False)
+    parser.add_argument('--flatmetric', type=string, default="inner", required=False)
```
Comment (lintool) [inline pyserini/index/faiss.py:38]: why `flatmetric` and not just `metric`?

**Review by lintool — COMMENTED** [review]
Hi, thanks for your contribution!

File: pyserini/index/faiss.py (hunk 1 above)
Comment (jonhilgart22) [reply pyserini/index/faiss.py:38]: both are specific for the flat index, but, likely, you should pick a metric and apply it to either the flat index or one of the other index builds.

Comment (lintool) [reply pyserini/index/faiss.py:38]: So we should rename to `--metric` then? @MXueguang @crystina-z thoughts?

**Comment by lintool** [comment]
@MXueguang what do we do with this PR?

**Review by MXueguang — APPROVED** [review]

**Pull request merged**